File: DSA/N_PARTH_PANDEY_590012761/Day9Question2.py
def count_vowels_and_consonants(s):
    vowels = "aeiou"
    vowel_count = 0
    consonant_count = 0

    for char in s.lower():
        if char.isalpha():
            if char in vowels:
                vowel_count += 1
            else:
                consonant_count += 1

    print(f"Vowels: {vowel_count}, Consonants: {consonant_count}")


count_vowels_and_consonants("hello")               
count_vowels_and_consonants("Python 3.8!")         
count_vowels_and_consonants("123")     

File: DSA/VikrantSinghBanwala_590018082/day9_question2.c
#include<stdio.h>
#include<ctype.h>
void cntVandC(char *str,int *vowelCount,int *consonantCount){
    *vowelCount=0;
    *consonantCount=0;

    while (*str){
        char ch=tolower(*str);
        if(isalpha(ch)){
            if(ch=='a' || ch=='e' || ch=='i' || ch=='o' || ch=='u'){
                (*vowelCount)++;
            }else{
                (*consonantCount)++;
            }
        }
        str++;
    }
}
int main(){
    char str1[]="hello";
    char str2[]="Programming";
    int vowels,consonants;
    cntVandC(str1,&vowels,&consonants);
    printf("Input:\"%s\"\nOutput:Vowels:%d,Consonants:%d\n",str1,vowels,consonants);
    cntVandC(str2,&vowels,&consonants);
    printf("Input:\"%s\"\nOutput:Vowels:%d,Consonants:%d\n",str2,vowels,consonants);
    return 0;
}
